docs/source/using_gridappsd/index.rst

```diff
@@ -69,7 +69,7 @@ Services
 The `GridAPPSD's Sensor Simulator` simulates real devices based upon the magnitude of "prestine" simulated values.
 Currently the sensor service supports angle and magnitude measurements.
 
-'_GridAPPS-D DNP3 Service <https://gridappsd-dnp3.readthedocs.io/en/develop/>'__ 
+'_GridAPPS-D DNP3 Service <https://gridappsd-dnp3.readthedocs.io/en/develop/>'
 The `GridAPPS-D DNP3 Service` integrates GridAPPS-D with DNP3 based commercial tools to enable the CIM and DNP3 data exchange of the devices.
```

Comment:
You need the _ in order for it to become a link...why is the _ in the front of the title...Follow the Sensor Simulator link above.